tests/entities/test_trace_info.py

```diff
@@ -1,11 +1,14 @@
+import pytest
+
```

Comment:
Can we add a test case where we do `TraceInfo.from_dict(TraceInfo.from_proto(...).to_dict())` to make sure that loading a trace from proto and converting to dict & back works properly? This will prevent recurrence of the issue.